test/RewardsStreamerMP.t.sol

```diff
@@ -1217,23 +1301,24 @@ contract UnstakeTest is StakeTest {
                 totalStaked: stakeAmount,
                 totalMP: (stakeAmount + expectedBonusMP) + stakeAmount, // we do `+ stakeAmount` we've accrued
                     // `stakeAmount` after 1 year
-                totalMaxMP: 52_465_753_424_657_534_246,
+                totalMaxMP: _calculateMaxTotalMP(stakeAmount, lockUpPeriod),
                 stakingBalance: 10e18,
                 rewardBalance: 0,
                 rewardIndex: 0
             })
         );
-
+        uint256 newBalance = stakeAmount - unstakeAmount;
         // unstake half of the tokens
         _unstake(alice, 5e18);
```

Comment:
If we introduce `unstakeAmount`, then it should also be used here in `_unstake()`